README.md

```diff
@@ -425,6 +433,12 @@ docker load -i /tmp/klimaschutzmonitor-dbeaver.tar
     mv docker-compose.yml docker-compose.yml.bak && cp /tmp/docker-compose.yml .
     docker-compose up --detach --no-build
     ```
+10. Install certificate renewal cron job:
```

Comment:
Would it be good to document, how to initially issue / install the cert here, as well? Or would that be done by acme.sh?